packaging/rpm/microshift.spec

```diff
@@ -217,6 +217,7 @@ fi
 sed -i -n -e '/^OPTIONS=/!p' -e '$aOPTIONS="--no-mlockall"' /etc/sysconfig/openvswitch
 %systemd_post microshift-ovs-init.service
 systemctl is-active --quiet NetworkManager && systemctl restart --quiet NetworkManager || true
+systemctl is-active --quiet openvswitch && systemctl restart --quiet openvswitch || true
```

Comment:
Should we enable the services here instead of the blueprint?

Reply:
We can add the following
`systemctl is-enabled --quiet openvswitch && systemctl enable --quiet openvswitch`

Reply:
Done